scripts/mainnet/reset/rebuild-spec.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -x
+
+NODE_BUILD=$1
```

Comment:
This argument should be the whole tag (currently would be `bmaas/nodefcl:<something>`). One less thing to change when we move where we're pushing images to.

Reply:
I agree, but then we get a problem with the `substitute_` functions, there is some coupling here between what the value is in the `docker-compose.yml` to be replaced (at the bmaas/nodefcl tag is also present) there. Maybe leave this in and just do a search replace over multiple files. Will revisit this tomorrow.